static/docs/commands-reference/run.md

```diff
@@ -146,6 +146,12 @@ pipeline.
 
   > The `dvc status` command will mention that the file is `not in cache`.
 
+- `--outs-persist` - the same as `-o` except that outputs will
```

Comment:
Maybe just 'the same as `-o` except that outputs will not be removed upon repro'? I'm not so sre about the "after multiple execution" part, it seems confusing.